packages/generators/generators/lib/files/js/plugin/server/routes/admin/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const myRoute = require('./my-route');
+
+module.exports = {
+  type: 'admin',
+  routes: [myRoute],
```

Comment:
Hey thanks for making the previous change. I should have been more specific sorry. Before you were spreading an existing array into a new array, so it was making a shallow copy of an array of objects into a new array of objects. Here you now have an array with an array of objects inside. My suggestion was to do something like this since there is only one set of routes:

```suggestion
module.exports = {
  type: 'admin',
  routes: myRoute,
```
Let me know what you think, sorry for the confusion.